components/add_groups_to_team_modal/add_groups_to_team_modal.test.jsx

```diff
@@ -56,7 +58,7 @@ describe('components/AddGroupsToTeamModal', () => {
         expect(wrapper.state('addError')).toEqual(message);
     });
 
-    /*test('should match state when handleSubmit is called', async () => {
+    test('should match state when handleSubmit is called', async () => {
         const linkGroupSyncable = jest.fn().
             mockResolvedValueOnce({error: true}).
             mockResolvedValue({data: true});
```

Comment:
See above for explanation of this change
```suggestion
        const linkGroupSyncable = jest.fn().mockResolvedValue({error: true, data: true});
```